backend/tests/test_schwab_csv_import.py

```
@@ -435,3 +452,168 @@ def test_import_does_not_leak_exception_message(self, client):
+        assert row["unit_amount"] == pytest.approx(25.00, abs=1e-2)
+        # A dividend moves no shares.
+        assert row["quantity"] == 0
+        assert row["premium"] == 0.0
+        assert row["strike"] is None
+        assert row["expiration"] is None
+        # Raw Schwab sub-type verbatim for future tax-bucket reporting.
+        assert row["close_reason"] == "Cash Dividend"
+
+    @pytest.mark.unit
+    @pytest.mark.parametrize(
+        "row,expected_reason",
```

Ruff (0.15.17)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)